sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1881,8 +1881,24 @@ pi_result piMemBufferCreate(pi_context Context, pi_mem_flags Flags, size_t Size,
                              &Ptr));
   }
 
-  if ((Flags & PI_MEM_FLAGS_HOST_PTR_USE) != 0 ||
-      (Flags & PI_MEM_FLAGS_HOST_PTR_COPY) != 0) {
+  if ((Flags & PI_MEM_FLAGS_HOST_PTR_ALLOC) != 0) {
```

Comment:
add comments what this is doing and explain how this is connected to "pinned" memory

Reply:
I am waiting on a query update. I'll modify it as soon as I have it.

Reply:
Is this complete now? Still, can you explain how this is going to be used? Shouldn't the 
implementation use allocated memory as the storage bits for the memory object, similar to PI_MEM_FLAGS_HOST_PTR_USE? 

What is the "pinned" memory extension exactly?

Reply:
@rbegam ping